releases/borg-1.1.rst

```diff
@@ -86,7 +86,16 @@ Compatibility notes for upgrading from Borg 1.0 to Borg 1.1:
   - the short form of --append-only (-a) has been removed.
   - running "borg init" via a "borg serve --append-only" server will *not* create
     an append-only repository any more. Use "borg init --append-only" to initialize an append-only repository.
-- borg create: the --exclude-if-present option now supports tagging a folder with any file system
+- borg create:
+
+  - A new option --files-cache MODE (default MODE is: ctime,size,inode) has been introduced and also the
+  default cache MODE has changed from "mtime,size,inode" in 1.0.x to "ctime,size,inode" in 1.1.x.
+  - **Be prepared that the first "borg create" after you upgrade to 1.1.x will run significantly slower than usual.**
+  This is due to a complete cache rebuild, caused by the changed cache mode.
+  Passing explicitly --files-cache with the old default: "mtime,size,inode" would avoid the slowdown,
+  but is not advisable as "ctime" is better as it reflects both file contents and metadata changes,
+  while "mtime" reflects only file contents changes.
+  - the --exclude-if-present option now supports tagging a folder with any file system
```

Comment:
thanks for the addition!

https://github.com/DrupaListo-com/borgbackup.github.io/blob/51182d9ee4a1d04d4f9966228fa8207952874049/releases/borg-1.1.rst

formatting looks not quite correct yet.

Reply:
Strange, I took the orig file and tried to mimic the formatting style... will look into it and fix it now.